packages/rps/README.md

```diff
@@ -64,26 +64,28 @@ This will fire up the Storybook panel inside a browser.
     ```
     yarn deployContracts
     ``` 
-    
+
+NOTE: The wallet also has to deploy the simple adjudicator on the same network.
```

Comment:
I don't think this is true (I was able to deploy `rps` contracts without the wallet contracts deployed.)

Reply:
Is the point that *funding a game* will not work unless wallet and app are pointing to the same network?

Reply:
On reflection, what I wrote is not true. I got into a state where the wallet hadn't deployed contracts to the same network, but it wasn't the adjudicator.

Reply:
```suggestion
```